Sources/Gtk/Widgets/Widget.swift

```diff
@@ -200,29 +200,26 @@ open class Widget: GObjectRepresentable {
         signals.append((handlerId, box))
     }
 
-    public func setForegroundColor(color: Color) {
-        let className = String("class-\(UUID().uuidString)").replacingOccurrences(
-            of: "-",
-            with: "_"
-        )
-        className.withCString { string in
-            gtk_widget_add_css_class(widgetPointer, string)
+    /// A block of CSS for this widget
+    public lazy var css: CSSBlock = CSSBlock(forCssClass: customCssClass) {
```

Comment:
Does this need to be lazy? As far as I understand that would only be avoiding initialising a very simple structure (as opposed to the lazy on cssProvider which is necessary to avoid making and attaching unused providers)

Reply:
Usually I only use `lazy` if I feel it really has a purpose (otherwise I'd be adding it everywhere for micro-optimisations that probably won't do much), so I'd so best to only have `lazy` for the provider

Reply:
actually -- the ObjectIdentifier init references self, which is only available in lazy so I will keep this lazy